index.html

```diff
@@ -187,26 +187,27 @@ <h2 class="about-grid-item-title">Skills</h2>
         </div>
       </div>
     </div>
-    <div id="contact-div">
-      <div id="contact-center">
-        <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
-        <div id="contact-form-div">
-          <form id = "survey-form">
-            <input id="fullname" class="form name" type="text" placeholder="Full Name" required>
-            <input id="first-name" class="form name desktop" type="text" placeholder="First Name" required><input id="last-name" class="form name desktop" type="text" placeholder="Last Name" required><br>
-            <input id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
-            <textarea id="comments" class="form" name="comments" placeholder="Hey I’d like to connect to..." required></textarea><br>
-            <div id="submit-button" class="grid-button-container">
-              <input type="submit" id="submit" class="grid-button" value="Get in touch">
-            </div>
-            <div id="email-para">
-              <p>nyldrift@hotmail.com</p>
-            </div>
-          </form>
+  </section>
+  <section id="contact">
+    <div id="contact-center">
+      <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
+      <div id="contact-form-div">
```

Comment:
- [ ] The contact form heading text in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=F1n7sMh4ZV6XntRp-0) is aligned in the centre of the container. Please make sure your heading also looks identical to that in the figma design as it is the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) that:

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

_Hint: Try adding `padding` or `margin` around the text_

your design:

<img width="309" alt="Screenshot 2023-03-29 at 8 48 33 AM" src="https://user-images.githubusercontent.com/44798044/228423642-3b34fe93-f20d-451a-9aa9-fb4aee9cf984.png">

figma design:

<img width="425" alt="Screenshot 2023-03-29 at 8 22 43 AM" src="https://user-images.githubusercontent.com/44798044/228423689-21d86ec2-48a8-4a7f-8036-5673aa772e1b.png">

Reply:
- the design was already like the figma design, i could not replicate the deformation seen in this printscreen,.